.claude/agents/gemini.md

```
@@ -5,14 +5,30 @@ tools: Bash, WebSearch, WebFetch, Read, Grep, Glob, LS
 color: blue
 ---
 
+**Before doing anything else**, verify Gemini is available by running this smoke test using the Bash tool:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)